pipeline/outputs/cloudwatch.md

```diff
@@ -17,6 +17,8 @@ This is the documentation for the core Fluent Bit CloudWatch plugin written in C
 | role\_arn | ARN of an IAM role to assume \(for cross account access\). |
 | auto\_create\_group | Automatically create the log group. Valid values are "true" or "false" \(case insensitive\). Defaults to false. |
 | endpoint | Specify a custom endpoint for the CloudWatch Logs API. |
+| metric\_namespace | A string representing the CloudWatch namespace for the metric. |
```

Comment:
Change to: "A string representing the CloudWatch namespace for metrics. See Host metrics tutorial below."

Otherwise its a bit confusing, because this is a Logs output but suddenly you're talking about metrics

Reply:
Updated.